client/packages/common/src/intl/currency/currency.test.tsx

```diff
@@ -42,12 +43,18 @@ describe('currency formatting - en', () => {
 });
 
 describe('currency formatting - fr', () => {
+  beforeAll(() => {
+    const { result: intlUtils } = renderHookWithProvider(useIntlUtils);
+    intlUtils.current.changeLanguage('fr');
```

Comment:
I get a warning that this needs to be wrapped in act() from `import { act } from 'react-dom/test-utils';`
```suggestion
    act(() => {
      intlUtils.current.changeLanguage('fr');
    });
```
afterwards it all runs fine